src/components/elements/default-tab.js

```diff
@@ -45,20 +45,20 @@ const TabButton = styled.button`
 const TabList = styled.div`
     display: flex;
     width: 100%;
-    justify-content: center;
+    justify-content: ${props => props.is_left_aligned ? 'left': 'center'};
```

Comment:
I don't think `left` is a valid value for justify-content, should be `flex-start`. Do you think it'll be better for us to pass in the value into the prop and set fallback as center such as below instead?
```
justify-content: ${(props) => props.jc ? props.jc : 'center'};
```
Or do you think for all the use cases, it should follow this sort of left alignment when passed in the prop.